### PR TITLE
New Programs added!

### PR DESCRIPTION
**Pull request by Sonichigo**
New program have been added as well as bit formatting is also done!

### CODE REVIEW

**Review by jcavenue — APPROVED** [review]
Look's Good to me.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@
 - <a href="https://www.cncf.io/people/ambassadors/">Cloud Native Ambassadors</a>
   - Cloud Native Ambassadors (CNAs) are individuals who are passionate about Cloud Native Computing Foundation technology and projects, recognized for their expertise, and willing to help others learn about the framework and community. This program is for everyone who love CNAs!
 
+- <a href="https://bit.ly/SAWO-Labs-Champ-Program">SAWO Champ Program</a>
+  - In SAWO Influencer Program, they aim to uplift our contributors who are adding value to the world using SAWO. To show their appreciation, everyone who contributes to SAWO Labs is eligible to receive some awesome benefits. This is a great opportunity for a developer and a content creator who provides recommendations to inspire their audience via social media.
```
Comment (Tine-Calumbiran) [inline README.md]: ### Need approval from SurPath Hub Admins
- Adding these lines is something that SurPath Hub admins need to decide within themselves. In the meantime, you may want to consider paraphrasing this with the same format as the other programs.

Other than that, everything looks good to me.

**Review by Sonichigo — COMMENTED** [review]
@chraem Changes done!

File: README.md [hunk 2]
```diff
@@ -83,3 +81,4 @@
 - [Hacktoberfest](https://hacktoberfest.digitalocean.com/)
 - [Radare Summer of Code](https://www.radare.org/rsoc/2019/)
 - [RAILS GIRLS SUMMER OF CODE](https://railsgirlssummerofcode.org/)
+- [SAWO Champ Program](https://bit.ly/SAWO-Labs-Champ-Program)
```
Comment (Tine-Calumbiran) [inline README.md:84]: Looks good to me 😊

**Review by Tine-Calumbiran — APPROVED** [review]

**Review by vinzvinci — APPROVED** [review]
LGTM 👍 Thanks everyone!

**Pull request merged**